packages/docs/src/lang/en/customization/SassVariables.json

```diff
@@ -1,8 +1,8 @@
 {
   "header": "# SASS variables",
   "headerText": "Starting with version **2.0**, Vuetify is built using **SASS/SCSS**. Using the data option in your `vue.config.js`, we can also pass in custom variables. A list of available variables is located in each components' folder. In order to build the SASS file, you will need to configure your application to support [SASS](https://sass-lang.com/dart-sass) and [Fibers](https://github.com/laverdet/node-fibers).",
-  "sassAlert1": "SCSS is used for variable files to support maps using indented syntax.",
-  "sassHeader2": "## Setup vue.config.js",
+  "sassAlert1": "Note: SCSS/SASS vrriables are only supported using [A-la-carte installation](https://vuetifyjs.com/en/customization/a-la-carte).",
```

Comment:
```suggestion
  "sassAlert1": "Note: SCSS/SASS vrriables are only supported using [A-la-carte installation](/customization/a-la-carte).",
```

Reply:
`variables` not `vrriables`

Reply:
Caught that after I accepted the suggestion, fix is already in the diff